bin/cli.js

```diff
@@ -23,7 +23,10 @@ const { readFile, writeFile } = require('./../lib/util/fs');
 const { listMigrations } = require('./utils/migrationsLister');
 
 async function openKnexfile(configPath) {
-  let config = require(configPath);
+  const importFile = require('../lib/util/import-file');
+  let config = await importFile(configPath);
+  // needs more testing
+  // config = config.default || config; //Allow default export || named exports
```

Comment:
should the commented out code be here?

Reply:
Well, I think is a missing feature.
Been thinking from the beginning on the 'default' export, 
but there is always more important TODO's, 
Didn't want to mix 'concern' and introduce unneeded changes
Can't make my mind up on that :)
SO I guess it should be removed